iot/manager/manager.go

```diff
@@ -613,6 +614,38 @@ func setConfig(projectID string, region string, registry string, deviceID string
 	return response, err
 }
 
+// sendCommand sends a command to a device listening for commands
+func sendCommand(projectID string, region string, registry string, deviceID string, sendData string) (*cloudiot.SendCommandToDeviceResponse, error) {
+	// [START iot_send_command]
```

Comment:
We're generally putting region tags outside the function. But, it's better to be consistent within a given product.

Reply:
Yeah for our other languages we have the region tags inside the functions. @gguuss any thoughts on this?

Reply:
I don't have a strong opinion on this -- AFAIK we're exposing these code snippets only within the context of static examples and are not composing snippets from multiple functions so either way they should appear in the documentation in a way that will work for our users.

Reply:
For Go, we've decided on region tags outside the function declaration. Some languages are inside, some are outside. But, if the other Go samples on this page have it a particular way, it's probably good to keep it consistent.

Reply:
I'll move the region tags outside since I need to refactor the other samples anyway to use `io.Writer` anyway